<launch>
    <node name="mapping" pkg="move_robot" type="mapping.py" output="screen">
        <param name="xlims" value="[0,20]"/>
        <param name="ylims" value="[0,20]"/>
        <param name="resolution" value="0.4"/>
        <param name="height" value="3"/>
        <param name="sigma_x" value="10"/>
        <param name="sigma_y" value="5"/>
        <param name="sensor_accuracy" value="0.8"/>
        <param name="verbose" value="false"/>
        
        # Sensor params
        <param name="fixed_frame" value="map"/>
        <param name="anemometer_frame" value="anemometer_frame"/>
        <param name="sensor_frame" value="mox_frame"/>
        <param name="use_service_for_gas" value="false"/>
    </node>
</launch>